### ISSUE TITLE
Выложить на PyPI

### ISSUE THREAD

**Issue by TitaniumHocker** [KlukvaMors/isdayoff#1]
Есть планы выложить в качестве пакета на PyPI? Чтоб можно было через `pip` ставить?

**Comment by KlukvaMors** [comment]
Да. Согласен нужно, пока никак руки не доходят, то тоска находит. Но постараюсь всё-таки предпринять действия в этом направление.

**Comment by asbevz** [comment]
Пересобрал новую версию: https://github.com/asbevz/isdayoff
